app/components/pages/SubmissionWizard/steps/Files/index.js

```diff
@@ -22,10 +22,69 @@ const UPLOAD_MUTATION = gql`
   }
 `
 
+const DELETE_MUTATION = gql`
+  mutation UploadFile($id: ID!) {
+    removeUploadedManuscript(id: $id) {
+      id
+      files {
+        filename
+        type
+      }
+    }
+  }
+`
+
 function getProgress(loading, data) {
   return loading || !data.uploadProgress ? 0 : data.uploadProgress
 }
 
+const onUploadValidationError = (
+  formFunctions,
+  deleteFile,
+  manuscriptId,
+  errorMessage,
+) => {
+  formFunctions.setFieldError('files', errorMessage)
+  deleteFile({
+    variables: { id: manuscriptId },
+  }).then(({ data }) => {
+    formFunctions.setFieldValue('files', data.removeUploadedManuscript.files)
```

Comment:
So for now it just sets the field value to be an empty array so if you navigate back then forward in the wizard you'll see the empty field state for the `Dropzone` rather than a 'Success' message. I suppose I could hardcode a `[]` but then if in the future someone introduces a bug in the mutation, the client would be out of sync with the data stored server side.

Reply:
Is this to support multiple file upload?

Reply:
If so, it's worth adding a test for this